lib/roo_on_rails/railties/sidekiq.rb

```diff
@@ -28,6 +29,20 @@ def config_sidekiq
         end
       end
 
+      def config_sidekiq_metrics
+        begin
+          require 'sidekiq/middleware/server/statsd'
+        rescue LoadError
+          return # metrics are only available in sidekiq pro
```

Comment:
Yeah I was debating that, but as this is an open-source library not everybody will have Sidekiq Pro. Then again, using non-Pro for commercial stuff is a bad idea as it can lose jobs, so maybe it's a reasonable warning anyway.